platform/security/security-migratable/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- /**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/ -->
+<blueprint xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
+           xsi:schemaLocation="http://www.osgi.org/xmlns/blueprint/v1.0.0 http://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd">
+
+    <bean id="migratableUtil" class="org.codice.ddf.migration.util.MigratableUtil"/>
+
+    <bean id="securityMigratable"
+          class="org.codice.ddf.security.migratable.impl.SecurityMigratable">
+        <argument value="Exports Security system files"/>
+        <argument value="false"/>
+        <argument ref="migratableUtil"/>
+    </bean>
+
+    <service id="securityMigratableService" ref="securityMigratable"
+             auto-export="interfaces"/>
```

Comment:
@pklinef: What's out preferred way of exporting a service? Using `auto-export` or explicitly exporting it using the `interface` attribute?